pkgs/top-level/all-packages.nix

```diff
@@ -25026,7 +25026,7 @@ in
 
   android-file-transfer = libsForQt5.callPackage ../tools/filesystems/android-file-transfer { };
 
-  antimicro = libsForQt5.callPackage ../tools/misc/antimicro { };
```

Comment:
Please add the `antimicro` attribute to `pkgs/top-level/aliases.nix` with a message pointing people to `antimicroX`. You can see the file for other examples.

Reply:
Thanks!